cli/azd/pkg/project/framework_service_docker.go

```diff
@@ -14,9 +14,11 @@ import (
 )
 
 type DockerProjectOptions struct {
-	Path     string `json:"path"`
-	Context  string `json:"context"`
-	Platform string `json:"platform"`
+	Path      string `json:"path"`
+	Context   string `json:"context"`
+	Platform  string `json:"platform"`
+	ImageName string `json:"imageName"`
+	ImageTag  string `json:"imageTag"`
```

Comment:
Do we need "Image" here?

Reply:
We may want to consider having an image section, because we may also want to support other image properties.

Reply:
Not sure how we feel about it, but I ended up folding the name and tag portion into a single attribute `tag` to be more consistent with docker. I also realized the conundrum that we already have image related properties on `docker` itself -- like the image arch, so this doesn't break consistency.